apps/sms/views/conversation/js/compose.js

```diff
@@ -420,8 +420,6 @@ var Compose = (function() {
         // Append each fragment in order to the composer
         Compose.append(fragment);
       }, Compose);
-
-      this.focus();
```

Comment:
Sounds good, but we don't pass ```focusComposer``` when we open thread with draft either from Inbox or notification, hence input is not focused when it should :)

Could you please add tiny integration test for this case as well, once you fix it?